db/migrate/20240418233828_create_accredited_representative_portal_verified_representatives.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CreateAccreditedRepresentativePortalVerifiedRepresentatives < ActiveRecord::Migration[7.1]
+  def change
+    create_table :accredited_representative_portal_verified_representatives do |t|
+      t.string :ogc_registration_number, null: false
+      t.string :first_name
```

Comment:
Is there any reason to have the first/last/middle names here? If you're going to link based on email then you prob don't even need these fields